examples/protocol_specification_ex/default.yaml

```diff
@@ -0,0 +1,12 @@
+name: default
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for exchanging any bytes message.'
+speech_acts:
+  bytes:
+    content: bytes
+  error:
+    error_code: ErrorCode
+    error_msg: str
+    error_data: dict
```

Comment:
@5A11 for version control its good to always add a blank line at the end of each file

Reply:
Done